### ISSUE TITLE
Settings: Clearer label on mobile theme option

### ISSUE THREAD

**Issue by beaulebens** [Automattic/jetpack#7036]
Right now, we have a label for the mobile theme that looks like this:

<img width="751" alt="screen shot 2017-04-22 at 2 06 42 pm" src="https://cloud.githubusercontent.com/assets/108942/25307807/ee23c52c-2764-11e7-82ec-5a1d334ec59d.png">

A few people have remarked that the main label is not very descriptive, so we should probably reword it to something like:

"Enable optimized mobile theme"

or

"Use an optimized theme for mobile devices"

or

"Use a mobile-specific theme for smart phones"

cc @rickybanister / @richardmuscat for wording suggestions.

**Comment by richardmuscat** [comment]
I would suggest simply going with "Use mobile theme."

**Comment by csonnek** [comment]
What came out of our discussion was to mirror the wording for the WordPress.com toolbar on the same page: 

"Enable the Jetpack Mobile theme"

It would keep instructions cleaner and consistent in the settings screens.

**Comment by rickybanister** [comment]
That sounds great @csonnek 

Whoever ends up hitting this, if they could make the same change in Calypso, that would be much appreciated.
